src/pages/organizations/[organizationid].js

```diff
@@ -158,9 +158,15 @@ export default function Organization(props) {
             <Box sx={{ mt: 2 }}>
               <Info
                 iconURI={CalendarIcon}
-                info={`Organization since ${moment(
-                  organization?.created_at,
-                ).format('MMMM DD, YYYY')}`}
+                info={
+                  <time
+                    dateTime={moment(organization?.created_at).format(
```

Comment:
No need to format the time for the prop value we can just use `organization.created_at`

Reply:
ok, I removed formate.